modin/core/storage_formats/cudf/parser.py

```diff
@@ -70,8 +70,8 @@ def get_dtypes(cls, dtypes_ids):
         )
 
     @classmethod
-    def single_worker_read(cls, fname, **kwargs):
-        ErrorMessage.default_to_pandas("Parameters provided")
+    def single_worker_read(cls, fname, *, reason, **kwargs):
```

Comment:
```suggestion
    def single_worker_read(cls, fname, reason=None, **kwargs):
```
Why is there an additional star here?